### PR TITLE
[WebUI] Add the first implementation of HatoholModal.

### PR DESCRIPTION
**Pull request by kz0817**
We will replace HatoholDialog that uses jQuery-UI with
HatoholModal by bootstrap's Modal. This patch is a mimimum
implementation to realize a reconnect modal, which will be
merged later.

### CODE REVIEW

File: client/static/js/hatohol_modal.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  }
+
+  var body = $('<div class="modal-body" />');
+  content.append(body);
+  body.append(params.body);
+  self.body = body;
+
+  if (params.footer != null) {
+    var footer = $('<div class="modal-footer" />');
+    content.append(footer);
```
Comment (cosmo0920) [inline client/static/js/hatohol_modal.js]: It seems to be `footer.append(params.footer);` is missing here. 
Is this intended?

Comment (kz0817) [inline client/static/js/hatohol_modal.js]: Oh. You're right. It's a mistake.
I'll fix it.

Comment (cosmo0920) [inline client/static/js/hatohol_modal.js]: Thanks for the confirming!
I'll rebase and send HatoholModal browser test patch after this PR is merged.

**Comment by kz0817** [comment]
I send the fixed version as #1829 and close this issue.

**Pull request closed without merging**